lib/declare_schema/model/foreign_key_definition.rb

```diff
@@ -15,10 +17,10 @@ def initialize(model, foreign_key, options = {})
         @child_table = model.table_name # unless a table rename, which would happen when a class is renamed??
         @parent_table_name = options[:parent_table]&.to_s
         @foreign_key_name = options[:foreign_key]&.to_s || @foreign_key
-        @index_name = options[:index_name]&.to_s || model.connection.index_name(model.table_name, column: @foreign_key_name)
```

Comment:
This `@index_name` was never used except in the lines below. So I just folded it in there.